pkg/network/encoding/usm.go

```diff
@@ -168,8 +168,6 @@ func (bc *USMConnectionIndex[K, V]) Close() {
 
 		telemetry.NewMetric(
 			fmt.Sprintf("usm.%s.orphan_aggregations", bc.protocol),
-			telemetry.OptMonotonic,
```

Comment:
why removing monotonic?

Reply:
I'm only simplifying things. Counters are now always monotonic. We never had the need for a non-monotonic counter, so I'm removing some cruft related to this. I plan to explicitly mention in the documentation once I make the PR officially ready for review